data/ocr/logo_annotation_labels.txt

```diff
@@ -1,3 +1,5 @@
 label rouge||en:label-rouge
+label rouge vectoriel||en:label-rouge
```

Comment:
@teolemon What is this "label rouge vectoriel" ?

Reply:
some less frequent variation I found in an OCR file